kentikapi/client.go

```diff
@@ -228,3 +260,23 @@ func (c Config) makeAuthInterceptor() grpc.UnaryClientInterceptor {
 		return invoker(ctx, method, req, reply, cc, opts...)
 	}
 }
+
+func (c Config) makeRetryInterceptor() grpc.UnaryClientInterceptor {
+	opts := []grpc_retry.CallOption{
+		grpc_retry.WithBackoff(grpc_retry.BackoffExponential(*c.RetryCfg.MinDelay)),
+		grpc_retry.WithCodes(codes.Unavailable),
+		// TODO: ensure c.RetryCfg.MaxAttempts is positive or research other options (mpalczynski)
```

Comment:
Let's just change type of c.RetryCfg.MaxAttempts to _uint_, so that the compiler enforces sane values.

Reply:
Done